DDG4/plugins/Geant4ExtraParticles.cpp

```diff
@@ -159,36 +147,15 @@ void Geant4ExtraParticles::constructParticle(Constructor& ) {
 
 void Geant4ExtraParticles::constructProcess(Constructor& ctor) {
   G4ParticleTable::G4PTblDicIterator* ParticleIterator = ctor.particleIterator();
-#if G4VERSION_NUMBER < 940
-  if ( 0 == _scatter ) _scatter=new G4hMultipleScattering();
-  if ( 0 == _ionise ) _ionise=new G4hIonisation()
-  if ( 0 == _decay ) _decay=new G4Decay()
-#endif
   while((*ParticleIterator)()) {
     G4ParticleDefinition* pdef = ParticleIterator->value();
     G4ProcessManager* pmgr = pdef->GetProcessManager();
     if (pdef->GetParticleType() == "extra") {
       if (pdef->GetPDGCharge() != 0) {
-#if G4VERSION_NUMBER < 940
-        pmgr->AddProcess(_scatter, -1,  1, 1); // multiple scattering
-        pmgr->AddProcess(_ionise,  -1,  2, 2); // ionisation
-        pmgr->AddProcess(_decay,   -1, -1, 2); // decay
-#else
         pmgr->AddProcess(new G4hMultipleScattering(), -1,  1, 1); //multiple scattering
         pmgr->AddProcess(new G4hIonisation(),  -1,  2, 2); // ionisation
```

Comment:
So we leave G4hMultipleScattering and G4hIonisation? Should we open that this is to be checked if there is double counting?

Reply:
solved with comment of Alberto

Reply:
As far as I can tell these are not added otherwise to the particles:
```
Idle> /particle/select susy-chi_1^+
Idle> /particle/process/dump
G4ProcessManager:  particle[susy-chi_1^+]
[0]=== process[Transportation :Transportation] Active
  Ordering::             AtRest             AlongStep          PostStep
                    GetPIL/    DoIt    GetPIL/    DoIt    GetPIL/    DoIt
  Ordering::
  index                 -1:      -1:       2:       0:       5:       0:
  parameter             -1:      -1:       0:       0:       0:       0:
[1]=== process[Decay :Decay] Active
  Ordering::             AtRest             AlongStep          PostStep
                    GetPIL/    DoIt    GetPIL/    DoIt    GetPIL/    DoIt
  Ordering::
  index                  0:       0:      -1:      -1:       2:       3:
  parameter           1000:    1000:      -1:      -1:    1000:    1000:
[2]=== process[msc :Electromagnetic] Active
  Ordering::             AtRest             AlongStep          PostStep
                    GetPIL/    DoIt    GetPIL/    DoIt    GetPIL/    DoIt
  Ordering::
  index                 -1:      -1:       1:       1:       4:       1:
  parameter             -1:      -1:       1:       1:       1:       1:
[3]=== process[hIoni :Electromagnetic] Active
  Ordering::             AtRest             AlongStep          PostStep
                    GetPIL/    DoIt    GetPIL/    DoIt    GetPIL/    DoIt
  Ordering::
  index                 -1:      -1:       0:       2:       3:       2:
  parameter             -1:      -1:       2:       2:       2:       2:
[4]=== process[StepLimiter :General] Active
  Ordering::             AtRest             AlongStep          PostStep
                    GetPIL/    DoIt    GetPIL/    DoIt    GetPIL/    DoIt
  Ordering::
  index                 -1:      -1:      -1:      -1:       1:       4:
  parameter             -1:      -1:      -1:      -1:    1000:    1000:
[5]=== process[UserSpecialCut :General] Active
  Ordering::             AtRest             AlongStep          PostStep
                    GetPIL/    DoIt    GetPIL/    DoIt    GetPIL/    DoIt
  Ordering::
  index                 -1:      -1:      -1:      -1:       0:       5:
  parameter             -1:      -1:      -1:      -1:    1000:    1000:
```